### PR TITLE
Change the gke minor regex to use single backslash to escape since it's in a raw string.

### PR DESCRIPTION
**Pull request by richackard**

### CODE REVIEW

File: kubetest2-gke/deployer/build/gke_make.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ const (
 )
 
 var (
-	gkeMinorVersionRegex = regexp.MustCompile(`^v(\\d\\.\\d+).*$`)
+	gkeMinorVersionRegex = regexp.MustCompile(`^v(\d\.\d+).*$`)
```
Comment (amwat) [inline kubetest2-gke/deployer/build/gke_make.go:47]: let's reuse the regex at
https://github.com/kubernetes-sigs/kubetest2/blob/master/kubetest2-gke/deployer/version.go#L43

we already have a few unit tests for that

Comment (richackard) [reply kubetest2-gke/deployer/build/gke_make.go:47]: The capture group in that regex is a bit tricky to get information from

Comment (amwat) [reply kubetest2-gke/deployer/build/gke_make.go:47]: ACK, let's keep it simple here then.

**Review by amwat — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**